packages/zoe/tools/priceAuthorityRegistry.js

```diff
@@ -4,8 +4,6 @@ import { E } from '@agoric/eventual-send';
 import { makeStore } from '@agoric/store';
 import { assert, details } from '@agoric/assert';
 
-import '../exported';
-
```

Comment:
I think this should remain

Reply:
`yarn lint` and vsCode are happy without it. I'm not going to try to explain that.